addons/purchase_stock/models/purchase.py

```diff
@@ -67,25 +67,6 @@ def _onchange_company_id(self):
         if not(p_type and p_type.code == 'incoming' and (p_type.warehouse_id.company_id == self.company_id or not p_type.warehouse_id)):
             self.picking_type_id = self._get_picking_type(self.company_id.id)
 
-    # --------------------------------------------------
```

Comment:
Maybe add in the commit message that it's not needed anymore because it's done automatically